assets/app/scripts/filters/resources.js

```diff
@@ -282,4 +282,68 @@ angular.module('openshiftConsole')
 
       return itemsArray;
     };
+  })
+  .filter('isTroubledPod', function() {
+    var fiveMinutesAgo = moment().subtract(5, 'm');
```

Comment:
can't calculate this at filter creation time, or "fiveMinutesAgo" will be pinned to five minutes before the console loaded